### ISSUE TITLE
Editorial disagreements

### ISSUE THREAD

**Issue by eltonteb** [pelagios/recogito2#260]
I agree that we don't want to enable the situation where the owner of the document simply deletes annotations made by others. However, perhaps we could use some kind of flagging system: i.e. the owner notices an incorrect annotation; they highlight it as an issue; a message gets sent to the person who made the annotation to have another look at it.

**Comment by rsimon** [comment]
A text comment or reply perhaps? I agree a notfication system when a user is mentioned in a thread (along the lines of using '@{username}' on GitHub, or '+{username}' on GoogleDrive would be great, and in fact somewhere on the list. (More specifically, notifications for sharing invitations etc.) But realistically it is nothing that can still be achieved in the remaining time/budget. (Keep in mind that next year focus is on Peripleo. Recogito will be mostly supported "as-is" with very little wiggle room for feature additions.)

**Comment by ChiaraPalladino** [comment]
I don't know if this makes any sense, but do you know that collective annotations are managed in Perseids by duplicating the files, so that every user has his/her own place of annotation without interfering with the others? The result is that there are various versions of each text, but every intervention is recorded with the name of the responsible and the type of modification. Not sure if it would be feasible to be used in Recogito, and how computationally difficult it would be to have this type of feature.

**Comment by rsimon** [comment]
But that means everyone is working on their own copy of the text, rather than multiple people on the same, if I understand correctly? Whereas Recogito's primary case is for multiple people to work on the same "instance" of the document. (Every intervention is still linked to the user that makes it, and the time it's made at.)

I think both approaches are valid, and applicable in different scenarios. This may relate also to issue #259, where we've discussed the ability to "fork" another users document - essentially creating a copy (perhaps along with the existing annotations), but then the fork would be a separate document, with an independent group of collaborators.

**Comment by ChiaraPalladino** [comment]
actually in Recogito 1 people could delete anyone's annotation and it was okay anyway, I guess because annotating and georeferencing places is tendentially _less_ controversial than, for example, linguistic annotation. 

Maybe it would be useful to have both options, considering that there may be two cases here: 1, a team of people that does collaborative annotation, 2, single users that want to use and modify those annotations for their specific purposes. The latter applies particularly to the discusses cases of tags and comments, if everyone applies his/her own vocabulary to a certain extent.

**Comment by rsimon** [comment]
I've been discussing with Elton (still via E-Mail I think) that perhaps document Admins should have the possibility to delete contributions, along the same lines as a blog admin can delete posts. Another possibility is to allow deleting someone else's place mapping, but not comments. (In fact this is how it works right now in Recogito 2, too.)

But irrespective of that: I agree that taking an existing annotated document, and producing "derivative work", as it were, sounds like another good use case, and we haven't thought of this yet. Indeed it would work very well with a forking mechanism, as @efi has suggested.

**Comment by ChiaraPalladino** [comment]
My concern is particularly about commentary work, additional to simple named entity annotation. It doesn't require much effort to modify a place reference, but it may be problematic to make your vocabulary co-exist with someone else's vocabulary, in tags for example. To export everything in an understandable format would be a complete mess, even worse to get derivative work from it. Perhaps the fork would be the most suitable strategy here. Maybe some option like "delete all comments" or "delete all tags/all selected tags" would be useful in this case?

**Comment by rsimon** [comment]
Ok, I'll make a mental not (and later a GitHub issue ;-) for forking, I think it's a really good idea. However, it will be difficult to fit it into the dev plan, so don't expect it to happen any time soon. 

Specific deletes: that I should be able to add pretty quickly. (If you go to the settings page now, under "Delete" there's an option to delete all annotations on the document. It would be easy to add more specific "delete all comments" etc. features there.)

Once that is done, I could at least offer a workaround: 
- It's possible to download a backup of a document + all annotations
- It will be (soon) possible to restore that backup (in the 'Account Settings' area)

Theoretically, you could ask for a backup from someone's document. And then restore it on your account. This would make you the owner of the doc, but otherwise leave all other data (the annotations, plus provenance information) unchanged. Essentially, it would be the same thing as forking, just without  the explicit "forking link" in the system.

**Comment by rsimon** [comment]
Superseded by forking (#274), which will be implemented according to schedule.
